src/api/ingest/bleaching_serializer.py

```diff
@@ -172,20 +172,39 @@ def skip_field(self, data, field):
 
         return field in getattr(self, empty_fields)
 
+    def _get_original_benthic_attribute_value(self, data):
```

Comment:
I guess there's a couple things:

1. ValidationError messages in ingest should be more detailed and be focused on the User rather than a developer.  
2. It is likely worth making generic but not sure if it falls in the scope of this issue and should have a card for just, refer back to comment 1. :)